docs/structure.rst

```diff
@@ -0,0 +1,128 @@
+Overview of directories and files
+=================================
+
+-  ``docs/``: documentation
+
```

Comment:
It would be good to add some README file here to explain the techno used for the doc, give links to documentation concerning sphinx, rst, etc., and explain the organization of the files and what needs to be done if one wants to add a new file, etc.

Reply:
I have added one in `docs/README.md`. Tell me if it's OK with you.